examples/getting_started/pendulum.py

```diff
@@ -13,6 +16,23 @@
 )
 
 
+def custom_dynamic(states, controls, parameters, nlp):
+    Dynamics.apply_parameters(parameters, nlp)
+    q, qdot, tau = Dynamics.dispatch_q_qdot_tau_data(states, controls, nlp)
+
+    qdot_reduced = nlp["q_mapping"].reduce.map(qdot)
+    qddot = biorbd.Model.ForwardDynamics(nlp["model"], q, qdot, tau).to_mx()
+    qddot_reduced = nlp["q_dot_mapping"].reduce.map(qddot)
+
+    return vertcat(qdot_reduced, qddot_reduced)
```

Comment:
As a matter of simplicity, I think the vertcat should be done silently inside of the function that called custom_dynamic. The user would therefore be expected to return a list